FirebaseSessions/Tests/Unit/IdentifiersTests.swift

```diff
@@ -60,15 +68,16 @@ class IdentifiersTests: XCTestCase {
 
     let firstSessionID = identifiers.sessionID
     XCTAssert(isValidSessionID(identifiers.sessionID))
-    XCTAssert(identifiers.previousSessionID.count == 0)
+    XCTAssertNil(identifiers.previousSessionID)
 
     identifiers.generateNewSessionID()
 
     XCTAssert(isValidSessionID(identifiers.sessionID))
-    XCTAssert(isValidSessionID(identifiers.previousSessionID))
+    XCTAssert(isValidSessionID(identifiers.previousSessionID ?? ""))
```

Comment:
Can we do something like below?
```
let previousSessionID = identifiers.previousSessionID
XCTAssert(isValidSessionID(previousSessionID!))
```
With the above statement, the actual error might get muddled in.